src/app/pages/user-team-create-page/user-team-create-page.component.html

```diff
@@ -0,0 +1,32 @@
+<div id="content">
+  <ebi-header></ebi-header>
+
+  <section id="main-content-area" class="row padding-top-xsmall padding-bottom-xsmall" role="main">
+    <h2 class="padding-top-xlarge">Create new Team</h2>
+
+
+    <form novalidate [formGroup]="teamCreateForm" >
+      <!-- Team Details Fieldset. -->
+      <fieldset required class="large-form padding-bottom-large padding-top-large">
+        <label>Institute / Project Name *
+          <input type="text" required placeholder="Enter institue or project name" class="form-control" name="centreName" formControlName="centreName"/>
+          <span class="form-error" *ngIf="teamCreateForm.get('centreName').hasError('required')">
+        Yo, you had better fill this out, it's required.
```

Comment:
Could you change this text, please?
Something like this: Centre name field is mandatory.
The same for the description field, please.